src/coreclr/nativeaot/Runtime/xoshiro128plusplus.h

```diff
@@ -0,0 +1,131 @@
+#pragma once
```

Comment:
Should this be move to src\native\minipal and CoreCLR impl switched to it as well?